test/Log/LogServiceTest.php

```diff
@@ -302,6 +330,39 @@ public function testGetDefaultLog()
         $this->assertSame($logger, $logService->getDefaultLog());
     }
 
+    /**
+     * Test logger created by createLogger().
+     *
+     * @throws Exception
+     * @throws \ReflectionException
+     */
+    public function testCreateLogger()
+    {
+        $logService = $this->getLogService();
+
+        $logFilePath = $logService->getPath() . 'default.log';
+        $logFile = @fopen($logFilePath, 'a', false);
+        if ($logFile === false) {
+            $path = dirname($logFilePath);
+
+            if (! is_dir($path)) {
+                throw new \Exception('Directory for logging does not exist');
+            } else {
+                throw new \Exception('Failed to open logging file:' . $logFilePath);
+            }
+        }
+
+        $reflection = new \ReflectionClass($logService);
+
+        $createLogger = $reflection->getMethod('createLogger');
+        $createLogger->setAccessible(true);
+        $logger = $createLogger->invokeArgs($logService, [$logService->getDefaultFormat(), $logService->getDefaultPriority(), $logFile]);
+
+        $this->assertNotNull($logger);
+        $this->assertInstanceOf(\Zend_Log::class, $logger);
+        $this->assertSame($logger, $logService->getDefaultLog());
```

Comment:
This check fails, because it should. It is not the purpose of `createLogger` to keep references to loggers. So, if you call `getDefaultLog`, you will get a new logger, because as far as the class is concerned,  the default log hasn't been created yet. So, please get rid of this line and check if the logger behaves as expected, format and priority.

Reply:
Please check if the test is as it should be in my latest commit.